src/coreclr/tools/aot/ILCompiler.Build.Tasks/ILCompiler.Build.Tasks.csproj

```diff
@@ -9,12 +9,16 @@
     <OutputPath>$(RuntimeBinDir)/ilc-published/netstandard</OutputPath>
     <Configurations>Debug;Release;Checked</Configurations>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <!--
+      The SDK warns if a project targets a framework that doesn't support trimming and IsTrimmable is true.
+      We set IsTrimmable higher up in the Directory.Build.props files, so we reset it to false here.
+    -->
+    <IsTrimmable>false</IsTrimmable>
```

Comment:
So netstandard2.0 doesn't support trimming, at all? If so, should we set this globally in a targets file based on the TFM?

Reply:
Yes, we don't have trimming support for .NET Standard 2.0 (it's completely unannotated, so we can't have accurate analysis).

Reply:
Then shouldn't we disable this globally in a targets file for all projects?